litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -296,7 +281,7 @@ const DashboardPage: React.FC = () => {
               variant="body1"
               className={classes.modalBody}
             >
-              Reconfigure this dashboard with a different datasource or update
+              Reconfigure this dashboard with a different data source or update
```

Comment:
Please take this from the translation file

Reply:
Sure @whitetiger1399